Plugin.pm

```diff
@@ -982,8 +983,10 @@ sub QobuzGetTracks {
 		}
 
 		if (!_isReleased($album) ) {
+			my @dt = split(/-/, $album->{release_date_stream});
+			my $rDate = strftime(preferences('server')->get('shortdateFormat'), 0, 0, 0, $dt[2], $dt[1] - 1, $dt[0] - 1900);
```

Comment:
Please create a small helper function for this. Otherwise a fix will always need to be applied to two places.

Reply:
Done.